resources/assets/components/actions/SocialDriveAction/SocialDriveAction.js

```diff
@@ -206,6 +221,10 @@ SocialDriveAction.propTypes = {
   fullWidth: PropTypes.bool,
   link: PropTypes.string.isRequired,
   pageId: PropTypes.string,
+  /**
+   * This prop expects a React component to allow user customization of link prop's query string.
+   * @see /resources/assets/components/pages/VoterRegistrationDrivePage/Alpha/AlphaPage
+   */
   queryOptions: PropTypes.object,
```

Comment:
I _think_ a `PropTypes.element` would denote a React component here.
```suggestion
  queryOptions: PropTypes.element,
```